### DISCUSSION TITLE
How do you print a program in vsCode?

### DISCUSSION THREAD

**Discussion by dhfreinkel** [codefori/vscode-ibmi#734] [IBM i]
I don't see how to do this.
Do I need to print from SEU?
TIA.

**Comment by worksofliam** [comment]
Not possible right now.

How about something like this? https://marketplace.visualstudio.com/items?itemName=TimMathias.CopyAsHTML

**Comment by m-tyler** [comment]
Not in color but you could create an action that runs this command, `sbmjob cmd(STRSEU SRCFILE(&OPENLIB/&OPENSPF) SRCMBR(&OPENMBR) TYPE(&EXT) OPTION(6)) job(PRTSRC) OUTQ(xxxxxxxx)`. Where, xxxxxxxx, is the name of an attached printer to your IBM i.  

It has to be run with SBMJOB otherwise the output gets redirected to the IBM i Output view.

**Reply by m-tyler** [reply]
Better in-tool options might also include.

https://marketplace.visualstudio.com/items?itemName=pdconsec.vscode-print
https://marketplace.visualstudio.com/items?itemName=nobuhito.printcode
